src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/taglets/ThrowsTaglet.java

```diff
@@ -195,34 +195,34 @@ private Content throwsTagsOutput(Map<ThrowsTree, ExecutableElement> throwsTags,
     }
 
     private Map<ThrowsTree, Element> expand(ThrowsTree tag, Element e, Utils utils, BaseConfiguration configuration) {
+        // although we could use LinkedHashMap for a single mapping too, to emphasize
+        // that the order is important, a Map.of() would do just fine
         // basically, flatmap... @throws -> @throws*
         if (tag.getDescription().stream().noneMatch(d -> d.getKind() == DocTree.Kind.INHERIT_DOC)) {
             return Map.of(tag, e);
+        }
+        // @throws is the only tag where it currently makes sense
```

Comment:
I don't think it always makes sense, so I would weaken this to _where it may currently make sense_ or similar.

It can make sense if the entire description is `{@inheritDoc}`
I don't think it makes sense when `{@inheritDoc}` is part of a longer description, with additional text and tags alongside `{@inheritDoc}`. I would be inclined to specify that as a format restriction for the use of `{@inherotDoc}` in `@throws`.